core/templates/base-components/oppia-footer.component.css

```diff
@@ -6,13 +6,22 @@
 .oppia-footer-container {
   margin-left: auto;
   margin-right: auto;
-  max-width: 800px;
+  max-width: 1150px;
```

Comment:
Just to check, what's the motivation behind this specific value?

Reply:
I had to increase the value to fit the Oppia mission text. I think this amount works best because it helps us make the most of the space and keeps things nice and tidy.

Reply:
Yup, but how was 1150 determined (as opposed to 1120 or 1180, say)?

Reply:
I am not sure if I will be able to answer it. This looked a bit more similar to the Figma file provided and also it seemed to maintain a similar gap between the flex items.